contracts/amm/AmmDataProvider.sol

```diff
@@ -57,32 +57,31 @@ contract AmmDataProvider is IAmmDataProvider {
     }
 
     function getVirtualReserves(
-        uint64 seriesId,
+        ISeriesController.Series memory series,
         address ammAddress,
         uint256 collateralTokenBalance,
         uint256 bTokenPrice
     ) public view override returns (uint256, uint256) {
-        ISeriesController.Series memory series = seriesController.series(
-            seriesId
-        );
-
+        //We set wTokenBalance = 0
+        //
+        /*
         uint256 wTokenBalance = erc1155Controller.balanceOf(
             ammAddress,
             SeriesLibrary.wTokenIndex(seriesId)
         );
         if (series.isPutOption) {
             wTokenBalance = seriesController.getCollateralPerOptionToken(
-                seriesId,
+                series,
                 wTokenBalance
             );
-        }
+        }*/
 
         return
             getVirtualReservesInternal(
```

Comment:
we should consolidate `getVirtualReservesInternal` and `getVirtualReserves` back into a single `getVirtualReserves` method since there's no point in having two methods now